cs/common/Reflection/ReflectionHelper.cs

```diff
@@ -147,7 +147,7 @@ select new
                 method = method.MakeGenericMethod(actualTypes);
                 //Debug.WriteLine("\t specializing: " + method);
             }
-            //Debug.WriteLine("Method: {0} for {1} {2}", method, name, string.Join(",", parameterTypes.Select(t => t.ToString())));
+            // Debug.WriteLine("Method: {0} for {1} {2}", method, name, string.Join(",", parameterTypes.Select(t => t.ToString())));
```

Comment:
This is really helpful from time to time to debug things. Maybe I should allow for tracing? Do you know about standard ways of configuring tracing for C# libs?